sbt-pgp/src/main/scala/com/jsuereth/sbtpgp/PgpSigner.scala

```diff
@@ -13,6 +13,12 @@ trait PgpSigner {
   def sign(file: File, signatureFile: File, s: TaskStreams): File
 }
 
+object PgpSigner {
+  // This is used to synchronize signing to work around
+  // https://github.com/sbt/sbt-pgp/issues/168
+  private[sbtpgp] val lock = new {}
```

Comment:
You can also do `new Object` without making a subclass.

Reply:
Fixed in https://github.com/sbt/sbt-pgp/pull/171/commits/33aaf75d1224b7631996b641bfa0e8424dece0ef